src/value.rs

```diff
@@ -11,8 +11,27 @@ pub enum Value {
     Str(String),
     Object(HashMap<String, Value>),
     Array(Vec<Value>),
+    Bool(bool)
 }
 
+impl Value {
+    pub fn str(val: &str) -> Value {
```

Comment:
I'm not strictly *against* it, but what's the matter with these helper functions that save you from writing `to_owned`? Is this common practice? :sweat_smile: